runtimes/standalone/Cargo.toml

```diff
@@ -76,6 +77,9 @@ runtime-benchmarks = [
   "frame-system-benchmarking",
   "hex-literal",
   "kilt-launch/runtime-benchmarks",
+  "kilt-primitives/runtime-benchmarks",
+  "kilt-support",
```

Comment:
I think we only use `AlwaysVerify` from `kilt-support`, so it should work also removing this?